### PR TITLE
Made buttons responsive

### PR DESCRIPTION
**Pull request by jSunpayco**
Am currently not aware how to make the questions responsive as it is an image, which restricts how the words are being wrapped.

### CODE REVIEW

**Comment by eduardconstantin** [comment]
I'll have a look and see what it can be done with the image

File: components/Quiz.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function Quiz() {
   if (error) return <p>Oh no... {error.message}</p>;
 
   return (
-    <div className="p-10 mx-auto w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
+    <div className="py-10 px-5 sm:p-10 mx-auto w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
```
Comment (eduardconstantin) [inline components/Quiz.tsx:32]:
```suggestion
    <div className="py-10 px-5 sm:p-10 mx-auto w-5/6 sm:w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
```

File: components/QuizForm.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const QuizForm: React.FC<Props> = ({ isLoading, questionSet, handleNextQuestion
       <div className="relative h-40">
         <Image src={imgUrl} alt="question" width={1200} height={200} unoptimized loading="eager" />
       </div>
-      <ul className="flex flex-col gap-2 my-5 select-none px-12 h-96">
+      <ul className="flex flex-col gap-2 mt-5 mb-16 select-none md:px-12 px-0 h-max">
```
Comment (eduardconstantin) [inline components/QuizForm.tsx:32]:
```suggestion
      <ul className="flex flex-col gap-2 mt-5 mb-16 select-none md:px-12 px-0 h-max min-h-[250px]">
```

**Review by eduardconstantin — COMMENTED** [review]
I'll fix the image in another branch

File: components/Quiz.tsx (hunk 1 above)
Comment (eduardconstantin) [reply components/Quiz.tsx:32]: @jSunpayco don't forget about this one too

Comment (jSunpayco) [reply components/Quiz.tsx:32]: Ah sure, that definitely looks better

**Pull request merged**